docs/containers-policy.json.5.md

```diff
@@ -90,7 +90,8 @@ Scopes matching individual images are named Docker references *in the fully expa
 using a tag or digest. For example, `docker.io/library/busybox:latest` (*not* `busybox:latest`).
 
 More general scopes are prefixes of individual-image scopes, and specify a repository (by omitting the tag or digest),
-a repository namespace, or a registry host (by only specifying the host name).
+a repository namespace, or a registry host (by only specifying the host name)
+    or a wildcarded expression for matching all subdomains.
```

Comment:
This needs to be precise enough to allow users to construct that expression (e.g. what the wildcard is), and not suggest that things like `foo*.*.bar` are valid.

Is the extra indentation intentional? It usually has semantic meaning in Markdown.